bitswap/options.go

```diff
@@ -99,3 +99,11 @@ func WithTracer(tap tracer.Tracer) Option {
 		}),
 	}
 }
+
+func WithClientOption(opt client.Option) Option {
+	return Option{opt}
+}
+
+func WithServerOption(opt server.Option) Option {
+	return Option{opt}
+}
```

Comment:
This was an unrelated change and should be a separate PR, I'll raise it separately. TLDR though is that `bitswap.New` can take options for the client or the server, but right now the only way to construct them is if they're forwarded as regular Bitswap options, otherwise you have to construct the client and server yourself. This saves on duplication while being more explicit and was something I ran into here because a new client option was introduced.